ui/lib/core/addon/components/masked-input.hbs

```diff
@@ -36,7 +36,7 @@
   {{/if}}
   {{#if @allowCopy}}
     <CopyButton
-      @clipboardText={{@value}}
```

Comment:
FWIW I don't think this needs to be updated for `CopyButton` just `Hds::CopyButton` which was implemented in 1.16 

It doesn't hurt to format 🤔 - mostly an FYI!

Reply:
I checked before I made this backport that it was still failing for objects. Otherwise I'd love to not need this backport 😂

Reply:
Nice catch!